### ISSUE TITLE
SSL Certificate renewal

### ISSUE THREAD

**Issue by NickRO19** [bitwarden/server#472]
So I accidentally allowed my SSL certificate from "Lets's Encrypt" to expire. How do I renew without having to reinstall Bitwarden?
I tried command "#certbot renew" but said command not found

**Comment by kspearrin** [comment]
How did you originally generate your LE cert? With the Bitwarden installer?

**Comment by NickRO19** [comment]
Yes using the installer. Lets encrypt.

-------- Original Message --------

**Comment by kspearrin** [comment]
Then just restarting the Bitwarden server with the script should automatically check the cert for renewal.

**Comment by NickRO19** [comment]
I tried that but it gives an error. I double checked that the ports I selected for access are open. I can write out the exact error when I get home later today. Just leaving for work.
I really appreciate your help thus far though! Thank you so much!
-------- Original Message --------

**Comment by kspearrin** [comment]
It would be helpful if you provided the actual error.

**Comment by NickRO19** [comment]
Yes, I will. Later today when I get back home I will send exactly what it says. Thanks again

-------- Original Message --------

**Comment by NickRO19** [comment]
So I think I may have figured it out on my own, maybe this is a bug??? I had changed the default ports in the config file at install because I was already using those ports.
Also I didn't want to keep port 80 open for security reasons...
Just to test I gave Bitwarden server port 80, and that seemed to have worked.
Does Bitwarden specifically always need port 80? Or did I miss something when changing the port config?
Using the alternative ports I could access and sync just fine. Just wasn't renewing cert.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
